tests/goth/assets/goth-config.yml

```diff
@@ -0,0 +1,62 @@
+# Note: values of keys denoting paths are resolved relative to the directory
+# in which this file is located.
+# The tokens `~` and `~user` are also replaced by the corresponding users's
```

Comment:
```suggestion
# The tokens `~` and `~user` are also replaced by the user's
```

Reply:
This file is now (as of https://github.com/golemfactory/yapapi/commit/f16cc5af2c76c3bfff91e38113d1a55e8226956e) generated by `goth` in a separate workflow step.